fatools/lib/fautil/mixin2.py

```diff
@@ -295,7 +295,7 @@ def align(self, parameters=None):
         self.scan(parameters)
 
         ladder = self.get_ladder_channel()
-        ladder.align(parameters)
+        ladder.align(parameters, None, None, self.get_saturated_rtimes())
```

Comment:
Not sure why, but i could not call this  like ...(parameters, saturated_rtimes=self.get_saturated(rtimes()).  Any idea why?